src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/inference/SimpleNovelAdjacencyInterpreter.java

```diff
@@ -50,7 +50,7 @@
                         .mapToPair(simpleNovelAdjacency ->
                                 new Tuple2<>(simpleNovelAdjacency,
                                         inferSimpleOrBNDTypesFromNovelAdjacency(simpleNovelAdjacency,
-                                                referenceBroadcast.getValue(), referenceSequenceDictionaryBroadcast.getValue())));
```

Comment:
Do the changes in this file have anything to do with the bug fix? Seems like you might want to save this to batch up with other related commits?

Reply:
nope. will revert back